store/extensions/StickiesArt/extension.json

```
@@ -0,0 +1,34 @@
+{
+    "title": "StickiesArt",
+    "description": "Write letters or phrases with stickies in your room or rooms where you have permission",
+    "authors": [
+      {
+        "name": "DanielaNaomi",
```

```suggestion
        "name": "DanielaNaomi"
```

The comma shouldn't be there 

(Also note: You don't need to make a new Pull Request when you make changes, any changes pushed to your branch will automatically get added to the existing Pull Request)